app/models/npc.rb

```diff
@@ -0,0 +1,51 @@
+class Npc < ActiveRecord::Base
+  class ConsequenceLimit < ActiveModel::Validator
+    def validate(npc)
+      if npc.consequences.length > 4
```

Comment:
Careful of magic numbers. the `4` is important state information to the system. You may wish to use an Enum style constant (or type) rather than these raw numbers.